presto-spi/src/main/java/com/facebook/presto/spi/utils/Utils.java

```diff
@@ -63,4 +63,11 @@ public static void checkState(boolean test)
             throw new IllegalStateException();
         }
     }
+
+    public static <T> T getOnlyElement(List<T> list)
```

Comment:
I am not sure if we want to make SPI to have such utility methods, as pointed out by @rongrong